ground-station-openmct/terminal/terminal_logic.js

```diff
@@ -43,8 +43,9 @@ form.addEventListener("submit", function (event) {
       return response;
     })
     .then((response) => {
-        const responseText = response.ack !== undefined ? `${response.message}${response.ack}` : response.message;
-        terminal.innerHTML += `$ ${response.message}\n`;
+        if (response.ack !== undefined) {
+            terminal.innerHTML += `$ ${response.message + response.ack}\n`;
```

Comment:
Can you change all the innerHTML to innerText. Daryl left this review earlier and I think innerText is more appropriate here